com.github.OpenHV.yaml

```diff
@@ -0,0 +1,63 @@
+app-id: com.github.OpenHV
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: openhv
+rename-desktop-file: openhv.desktop
+rename-appdata-file: openhv.metainfo.xml
+rename-icon: openhv
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet6
+finish-args:
+  - --socket=wayland
```

Comment:
does it really support wayland?

Reply:
You tell me. It is SDL2 based.

Reply:
it doesn't:

```
[bpiotrowski@risuku ~]$ flatpak run io.github.openhv.OpenHV
Platform is Linux
Engine version is e685731b337d76ba4d03b7dc84105029870236d4
Runtime: .NET CLR 6.0.5
Renderer initialization failed. Check graphics.log for details.
Renderer initialization failed. Check graphics.log for details.
Exception of type `System.InvalidOperationException`: Failed to initialize platform-integration library. Check graphics.log for details.
   at OpenRA.Game.Initialize(Arguments args) in /run/build/OpenHV/engine/OpenRA.Game/Game.cs:line 347
   at OpenRA.Game.InitializeAndRun(String[] args) in /run/build/OpenHV/engine/OpenRA.Game/Game.cs:line 290
   at OpenRA.Launcher.Program.Main(String[] args) in /run/build/OpenHV/engine/OpenRA.Launcher/Program.cs:line 32
```

Reply:
> You tell me. It is SDL2 based.

if I ask the question, it's because I don't have the answer.

Reply:
Check `~/.var/app/com.github.OpenHV/config/openra/Logs/graphics.log`. Maybe it is something stupid that can be easily fixed.